contracts/bridge/RateLimiter.sol

```diff
@@ -8,8 +8,8 @@ import "@openzeppelin/contracts-4.3.1-upgradeable/utils/math/MathUpgradeable.sol
 
```

Comment:
Should we bump up all OZ to latest version of 4.5.0-upgradeable? I started using it on feature/messaging.

Reply:
Fixed in [05ed501](https://github.com/synapsecns/synapse-contracts/pull/126/commits/05ed501d21cb8b3ee873ce3656501874c6babbec)